core/src/main/java/com/csi4999/screens/GenerationScreen.java

```diff
@@ -0,0 +1,234 @@
+package com.csi4999.screens;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.Align;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+import com.csi4999.ALifeApp;
+import com.csi4999.singletons.CustomAssetManager;
+import com.csi4999.singletons.ScreenStack;
+import com.csi4999.systems.environment.EnvProperties;
+import com.csi4999.systems.networking.GameClient;
+import com.csi4999.systems.networking.clientListeners.DescriptionListener;
+import com.csi4999.systems.networking.packets.RequestSavedEntityDataPacket;
+import com.esotericsoftware.kryonet.Client;
+
+import static com.csi4999.singletons.CustomAssetManager.SKIN_MAIN;
+import static com.csi4999.singletons.CustomAssetManager.UI_FONT;
+
+public class GenerationScreen implements Screen {
+    private Skin skin;
+    private final OrthographicCamera menuCam;
+    private final FitViewport menuViewport;
+    private final ALifeApp app;
+    private Stage stage;
+
+    private EnvProperties properties;
+
+    public GenerationScreen(ALifeApp app) {
+        this.app = app;
+        this.properties = EnvProperties.makeTestDefault();
```

Comment:
makeTestDefault() shouldn't be used because it starts with some components that i arbitrarily decided. It was used to make a test EnvProperties but it shouldn't be used now. The default list of available tools and sensors should be constructed in the UserAccountPacket